### ISSUE TITLE
White Text over the Homescreen Image: Could we add a mask OR move the white text down that appears on the home screen over the image.

### ISSUE THREAD

**Issue by tgascoigne14** [CCI-Studios/Learninghub#277]
"It is difficult for some learners to read and does not align with our AODA requirements"

If you have a better suggestion please let me know.

**Comment by ccmorris** [comment]
@tgascoigne14 
I'm not clear about what they meant by 'mask', but I made two very different POCs to show some improvements. The first one I just made the background image darker. It is already darkened, but going even darker increases the contrast with the white text on top. The second option is a bigger change to give the text a solid background colour and therefore move it out from on top of the image.

1:
![poc2](https://user-images.githubusercontent.com/4417748/87579615-312a1880-c6a4-11ea-8c73-e271a2ba1d81.JPG)


2:
![poc](https://user-images.githubusercontent.com/4417748/87579628-35563600-c6a4-11ea-94a0-939eac7ae5e2.JPG)
